bee-network/bee-gossip/src/service/host.rs

```diff
@@ -213,37 +217,47 @@ async fn peerstate_checker(shutdown: Shutdown, senders: Senders, peerlist: PeerL
     // Check, if there are any disconnected known peers, and schedule a reconnect attempt for each
     // of those.
     while interval.next().await.is_some() {
-        let peerlist_reader = peerlist.0.read().await;
+        let read = peerlist.0.read().await;
 
         // To how many known peers are we currently connected.
-        let num_known = peerlist_reader.filter_count(|info, _| info.relation.is_known());
-        let num_connected_known =
-            peerlist_reader.filter_count(|info, state| info.relation.is_known() && state.is_connected());
+        let num_known = read.filter_count(|info, _, _| info.relation.is_known());
+        let num_connected_known = read.filter_count(|info, state, _| info.relation.is_known() && state.is_connected());
 
         // To how many unknown peers are we currently connected.
         let num_connected_unknown =
-            peerlist_reader.filter_count(|info, state| info.relation.is_unknown() && state.is_connected());
+            read.filter_count(|info, state, _| info.relation.is_unknown() && state.is_connected());
 
         // To how many discovered peers are we currently connected.
         let num_connected_discovered =
-            peerlist_reader.filter_count(|info, state| info.relation.is_discovered() && state.is_connected());
+            read.filter_count(|info, state, _| info.relation.is_discovered() && state.is_connected());
+
+        // How many peers we know of but are currently disconnected.
+        let num_disconnected = read.filter_count(|_, state, _| state.is_disconnected());
 
         info!(
-            "Connected peers: known {}/{} unknown {}/{} discovered {}/{}.",
+            "Connected peers: known {}/{} unknown {}/{} discovered {}/{} - Disconnected peers: {}.",
```

Comment:
```suggestion
            "Connected peers: known {}/{} unknown {}/{} discovered {}/{} disconnected {}.",
```
I think this would be more consistent

Reply:
```suggestion
            "Peers: known {}/{} unknown {}/{} discovered {}/{} disconnected {}.",
```

Reply:
I am willing to change the log output, but the suggested change is illogical to me:
"Connected peers: ....... disconnected {}." hurts my brain too much :joy: I am open to other suggestions though.

Reply:
Yeah Jochen's one is better

Reply:
So that in "{1}/{2}" 1 means "connected", and not "healthy", "sync", "in love with" is obvious?

Reply:
I still like the original much more. I don't see where it's inconsistent, and removing the "Connected" makes things a bit vague, although a 3 year old is probably right in guessing here.

Reply:
Okay, since you approved, I am closing this noting that I am not against a change at all, just think, that the suggested proposals have problems of their own :wink: